karapace/constants.py

```diff
@@ -8,7 +8,7 @@
 
 SCHEMA_TOPIC_NUM_PARTITIONS: Final = 1
 API_VERSION_AUTO_TIMEOUT_MS: Final = 30000
-TOPIC_CREATION_TIMEOUT_MS: Final = 20000
+TOPIC_CREATION_TIMEOUT_S: Final = 20
```

Comment:
suggestion (non-blocking): Since this is changing, I suggest using the unambiguous `datetime.timedelta` for this value.

Reply:
I'd leave this out for now, I think we can introduce timedeltas for these as a future improvement.